### PR TITLE
Build version 2020.01

### PR DESCRIPTION
**Pull request by moritz**
* Bump version to 2020.01
* Build based on Debian Buster, which has been stable for some time now (and avoids some GPG shenenigans)
* add the current Star maintainer's GPG key

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM alpine:3.10
 
 RUN addgroup -S perl6 && adduser -S perl6 -G perl6
```
Comment (JJ) [inline alpine/Dockerfile]: while we're at it, wouldn't it be better to change this to raku?

**Review by JJ — APPROVED** [review]
It's probably OK, but if you want, maybe you can move from perl6 to raku too everywhere.

**Comment by hoelzro** [comment]
@moritz LGTM, and I agree with @JJ's suggestion re: the rename

**Comment by moritz** [comment]
added the rename as well, merging now...

**Pull request merged**